### ISSUE TITLE
Offer ability to short circuit help output (Ex: -h)

### ISSUE THREAD

**Issue by Boscop** [google/argh#30]
Currently, passing `-h` doesn't work:
> Unrecognized argument: -h

Passing `-h` should work the same as `--help`, unless one of the options/switches uses `short = 'h'`.

---
And in the Usage output, it would then show 
> -h, --help            display usage information

instead of just
> --help            display usage information

**Comment by Boscop** [comment]
Alternatively, to let the programmer decided (because maybe they want to use `-h` for something else),
there should be a way to manually call the function that prints the usage information (that's called when `--help` is passed).
So that it can be manually called when `-h` is passed with an explicit `#[argh(switch, short = 'h')] help: bool` field.

**Comment by benbrittain** [comment]
I'd be far more amicable to that change! That seems like a good option for those that want `-h` support

**Comment by msrd0** [comment]
I'd also like to see an `-h` shorthand flag, and I'd be willing to create a PR. However, I am concerned that letting the user implement the shorthand and argh worry about the long flag, the help would end up looking something like that:

```
Options:
  -h                display usage information
  --help            display usage information
```

Therefore, I think either the user should be given the option to specify a shorthand for the help message, or be able to "replace" the help switch altogether. Let me know what you think.

**Comment by benbrittain** [comment]
oh interesting, yes that makes sense. I'm open to a change like that!

**Comment by icefoxen** [comment]
Agreed on the `-h` shorthand flag.  As for anything else, I don't want programmers to decide that `-h` should mean anything else, I want it to just be always a shorthand for `--help`.

**Comment by aminya** [comment]
I made a pull request that fixes this issue #106
